.github/workflows/generate-builds.yml

```diff
@@ -124,7 +128,7 @@ jobs:
           soh.appimage
           readme.txt
   build-switch:
-    runs-on: ${{ vars.LINUX_RUNNER || 'ubuntu-latest' }}
+    runs-on: ${{ (vars.LINUX_RUNNER && fromJSON(vars.LINUX_RUNNER)) || 'ubuntu-latest' }}
```

Comment:
we wouldn't need a third machine, we'd just need a third var

if we wanted to use a different machine that is more in line with whatever `ubuntu-latest` is at any given point we'd have that option, but we can also just set that var to be the same as either compat or perf

Reply:
it also may be possible (since these jobs use docker) to run this on either perf or compat, in which case we could have
`LINUX_RUNNER`: `['self-hosted', 'linux']`
`LINUX_PERFORMANCE_RUNNER`: `['self-hosted', 'linux', 'perf']`
`LINUX_COMPATIBILITY_RUNNER`: `['self-hosted', 'linux', 'compat']`